health/REFERENCE.md

```diff
@@ -413,7 +413,10 @@ warn: $this > (($status >= $WARNING)  ? (75) : (85))
 crit: $this > (($status == $CRITICAL) ? (85) : (95))
 ```
 
-The above say:
+The above is graphically illustrated in the image bellow. Let's assume that the alert's metric, for some unknown reason, oscillates around 85. *With hysteresis taken into account,the alarm will be activated  only once.*
+![](hysteresis.png)
```

Comment:
For images, we just host them on GitHub's CDN so you don't have to embed them in the repo itself. Here's a link to the image. You can add this into the Markdown, add an alt tag, and remove the .png itself.

![image](https://user-images.githubusercontent.com/1153921/70662401-61e43400-1c23-11ea-8ec1-7fc0be4ad49d.png)